src/utils/fullscreen.js

```diff
@@ -0,0 +1,114 @@
+import {document} from './globals';
```

Comment:
I prefer not having to include this in our library if we could just invoke Mapbox's native FullScreenControl.